Containerfile.stream9

```diff
@@ -1,8 +1,8 @@
-FROM quay.io/almalinuxorg/8-minimal:8
+FROM quay.io/centos/centos:stream9
 
 MAINTAINER CRC team
 
-RUN microdnf install -y libvirt-devel curl git gcc make golang diffutils
+RUN dnf --enablerepo=crb install -y libvirt-devel curl git gcc make golang diffutils
```

Comment:
Another option could be to keep using almalinux, and install go 1.23 ourselves from a tarball. Not sure it would be better though, just an alternative suggestion, feel free to ignore.

Reply:
Ignored :)